ingestion/src/metadata/ingestion/source/database/unitycatalog/queries.py

```diff
@@ -0,0 +1,19 @@
+"""
+SQL queries for Unity Catalog
+"""
+
+UNITY_CATALOG_GET_CATALOGS_TAGS = """
+SELECT * FROM `{database}`.information_schema.catalog_tags;
+"""
+
+UNITY_CATALOG_GET_SCHEMA_TAGS = """
+SELECT * FROM `{database}`.information_schema.schema_tags WHERE schema_name = '{schema}';
+"""
+
+UNITY_CATALOG_GET_TABLE_TAGS = """
+SELECT * FROM `{database}`.information_schema.table_tags WHERE schema_name = '{schema}' AND table_name = '{table}';
```

Comment:
also we should optimise this we do not want to execute a query for each table to extract tags and same for columns 

we should extract all the tags of tables within a schema and cache it (per schema) similar to how we do it for table comments